cpp/src/arrow/adapters/arrow-rados-cls/cls_arrow.cc

```diff
@@ -126,6 +155,15 @@ class RandomAccessObject : public arrow::io::RandomAccessFile {
   std::vector<ceph::bufferlist*> chunks_;
 };
 
+/// \brief Scan the Parquet file objects.
+/// \param[in] hctx Context.
+/// \param[in] filter Filters.
```

Comment:
We can say, the filter expression to apply. Small one-word descriptions diminished the reason of adding doc strings

Reply:
Done. I wasn't sure what the filters were used for, hence the vague description. Fixed it now.